$DragonFly: src/gnu/usr.bin/groff/tmac/patches/tty.tmac.no_obj.patch,v 1.1 2005/09/01 20:47:14 asmodai Exp $

--- tty.tmac	2005-08-31 14:28:07.000000000 +0200
+++ tty.tmac	2005-08-31 15:51:25.000000000 +0200
@@ -83,6 +80,9 @@
 .ie '\*(.T'cp1047' \
 .  do mso cp1047.tmac
 .el \
+.  ie '\*(.T'koi8-r' \
+.    do mso koi8-r.tmac
+.el \
 .  if !'\*(.T'ascii' \
 .    do mso latin1.tmac
 .
